pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -87,7 +110,7 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 	}
 }
 
-func (e *Executor) checkApproval(ctx context.Context) (string, bool) {
+func (e *Executor) checkApproval(ctx context.Context, num int) (string, bool) {
```

Comment:
`num` in this function and `validateApproverNum()` is relatively long-term variable, so I would prefer to be kind of like `minApproverNum`.